prac_06/guitars.py

```diff
@@ -0,0 +1,64 @@
+from prac_06.guitar import Guitar
+
+
+def main():
+    guitars = []
+    print('My guitars!')
+    print()
+    name = input('Name: ')
+    while name != '':
+        year = get_number('Year: ', 'Year must be a number', 'Year must be above 0', 0, 2018, 'int')
+        cost = get_number('Cost: ', 'Year must be a number', 'Year must be above 0', 0)
+        guitars.append(Guitar(name, year, cost))
+        print()
+        name = input('Name: ')
+
+    print()
+    longest_word = get_largest_name(guitars)
+    longest_cost = get_largest_cost(guitars)
+    for i, guitar in enumerate(guitars):
+        print("Guitar {0}: {self.name:>{1}} ({self.year}), worth ${self.cost:{2},.2f}{3}"
```

Comment:
Personally I find this hard to follow using default naming seems cleaner in this case.

Reply:
```python
    for i, guitar in enumerate(guitars):
        print(
            "Guitar {}: {} ({}), worth ${:0,.2f}"
            .format(i + 1, guitar.name, guitar.year, guitar.cost),
            end=""
        )
        if guitar.is_vintage():
            print(" (vintage)")
        else:
            print("")
```